_events/sr2023/london-tech-day-december.md

```diff
@@ -22,7 +22,7 @@ like to attend. We will be confirming places shortly after you sign up.
 At the very latest please signup by Wednesday 30th November 2022, though we may
 close signups sooner if all places are filled.
 
-At the request of the venue: paint, power tools or hand tools which create mess
+At the request of the venue: paint, hand tools which create mess or power tools
```

Comment:
Nit: Would this clear up the confusion about the "hand tools which create power tools"?
```suggestion
At the request of the venue: paint, hand tools which create mess, or power tools
```

Also happy for this to merged as is if you'd rather

Reply:
Ah, hrm. :thinking: I agree that solves the "hand tools which create power tools" but the grammar pedant inside me doesn't like it.

I think I'm going to ship without this, but paging @prophile in case he can come up with something better.

Reply:
Grammarly suggests its ok https://www.grammarly.com/blog/comma-before-or/

Would an alternative be to change the `or` to `and`

Reply:
1380f07c9e1917e594cf95971ec5aa72d6ea9fc2